docs/querying/api.md

```diff
@@ -733,6 +745,12 @@ Example:
 $ curl -X POST \
   -g 'http://localhost:9090/api/v1/admin/tsdb/delete_series?match[]=up&match[]=process_start_time_seconds{job="prometheus"}'
 ```
+
+```json
+$ curl -X PUT \
+  -g 'http://localhost:9090/api/v1/admin/tsdb/delete_series?match[]=up&match[]=process_start_time_seconds{job="prometheus"}'
+```
+
```

Comment:
You should add a note here that PUT was added in 2.9

Reply:
Done